context.go

```diff
@@ -150,7 +150,7 @@ func (s *Subscription) nextMsgWithContext(ctx context.Context, pullSubInternal,
 		// If internal and we don't want to wait, signal that there is no
 		// message in the internal queue.
 		if pullSubInternal && !waitIfNoMsg {
-			return nil, errNoMessages
+			return nil, ErrNoMessages
```

Comment:
No need because the `pullSubInternal` indicates that this is an internal call and that error will not be surfaced to the user.